### ISSUE TITLE
Scenario: highlight expandable nodes

### ISSUE THREAD

**Issue by tfjmp** [CamFlow/CamFlowR#6]
That probably should be a change for the main cytoscape-prov.js library.

**Comment by MKLau** [comment]
Yeah, this would be good. Something like an outline around expandable nodes seems intuitive.
